docker/chainnode/entrypoint.sh

```diff
@@ -316,17 +334,17 @@ exec ronin $params \
   --syncmode $syncmode \
   --verbosity $VERBOSITY \
   --datadir $datadir \
-  --port 30303 \
+  --port $NETWORK_PORT \
```

Comment:
Should use this one $network_port ?  Otherwise the logic is broken when `NETWORK_PORT` is not set ? 
Maybe u can use the other approach like this which is quite easier 
```
--port  ${NETWORK_PORT:-30303}
```

Reply:
fixed for all 3 new params